x-pack/plugins/observability_solution/apm/public/components/shared/add_data_buttons/buttons.tsx

```diff
@@ -40,28 +43,28 @@ export function AddApmData({
 }) {
   const { core } = useApmPluginContext();
   const { basePath } = core.http;
-  const {
-    application: { navigateToUrl },
-  } = useKibana().services;
 
-  function handleClick() {
-    navigateToUrl(basePath.prepend(addApmData.link));
-    onClick?.();
-  }
   return (
-    <EuiButton data-test-subj={props['data-test-subj']} size="s" onClick={handleClick}>
+    <EuiButton
+      data-test-subj={props['data-test-subj']}
+      size="s"
+      onClick={onClick}
+      href={basePath.prepend(addApmData.link)}
+    >
       {addApmData.name}
     </EuiButton>
   );
 }
 
 export function AssociateServiceLogs({ onClick }: { onClick?: () => void }) {
-  function handleClick() {
-    window.open(associateServiceLogs.link, '_blank');
-    onClick?.();
-  }
   return (
-    <EuiButton data-test-subj="associateServiceLogsButton" size="s" onClick={handleClick}>
+    <EuiButton
+      data-test-subj="associateServiceLogsButton"
```

Comment:
Since this is an exernal link it should have a popout indicator.

```suggestion
      data-test-subj="associateServiceLogsButton"
      iconType="popout"
      iconSide="right"
```